up42/__init__.py

```diff
@@ -131,20 +131,17 @@ def validate_manifest(path_or_json: Union[str, Path, Dict]) -> Dict:
 def read_vector_file(
     filename: str = "aoi.geojson", as_dataframe: bool = False
 ) -> FeatureCollection:
-    tools = Tools(auth=_auth)
-    return tools.read_vector_file(filename, as_dataframe)
+    return Tools().read_vector_file(filename, as_dataframe)
```

Comment:
Why are we instantiating the Tools object? If read_vector_file is a static method we can just do `Tools.read_vector_file()` and avoid creating the object.

Reply:
Needs the self for using `self._deprecate_tools` in the function. Which needs to be within the class as it requires access to `self.__class__.__name__`. Added as reminder to the todo to remove the instantiation when finally deprecated, thanks.